### PR TITLE
Moving Remove display and Arduino from getting started to docs page

### PR DESCRIPTION
**Pull request by rachitb777**

### CODE REVIEW

File: _data/_docsandtutorials.json [hunk 1]
```diff
@@ -313,11 +319,25 @@
          "tags":["advanced"]
       },
       {
-         "title":"Enabling Secure Boot and BitLocker",
-         "description":"Learn how to secure your IoT device by enabling Secure Boot and BitLocker device encryption on supported IoT Core devices",
-         "link-text":"<a href='SB_BL.htm'>Learn More</a>",
-         "ranking":10,
-         "tags":["advanced"]
+         "title": "Enabling Secure Boot and BitLocker",
```
Comment (alecont) [inline _data/_docsandtutorials.json:322]: why are these entries repeated twice?

Comment (rachitb777) [inline _data/_docsandtutorials.json:322]: One is for en-US and another for zh-CN

Comment (alecont) [inline _data/_docsandtutorials.json:322]: ah ok! looks good!

**Comment by alecont** [comment]
looks good. Ivor, can you review and merge?

**Pull request merged**